<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WTransmit</class>
 <widget class="QWidget" name="WTransmit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>32</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>32</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DLC</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Data</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Period, ms</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Count</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Comment</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Tx control</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,1,0,0,0,0,0" columnminimumwidth="0,0,0,0,0,0,0,0,0,0,0">
        <property name="topMargin">
         <number>12</number>
        </property>
        <item row="1" column="4">
         <widget class="QCheckBox" name="rtr_chb">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="9" rowspan="2">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Data</string>
          </property>
         </widget>
        </item>
        <item row="0" column="10">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Comment</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="id_le">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="inputMask">
           <string>HHHHHHHH</string>
          </property>
          <property name="maxLength">
           <number>8</number>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="bits29_chb">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="10">
         <widget class="QLineEdit" name="comment_le"/>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>RTR</string>
          </property>
         </widget>
        </item>
        <item row="1" column="8">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="data1_le">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>HH</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="data2_le">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>HH</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="data3_le">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>HH</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="data4_le">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>HH</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="data5_le">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>HH</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="data6_le">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>HH</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="data7_le">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>HH</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="data8_le">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>HH</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>29Bits</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>DLC</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5" rowspan="2">
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QLineEdit" name="dlc_le">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="inputMask">
           <string>9</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="2">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="7" rowspan="2">
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineEdit_12">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>1000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Periodic_pb">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Periodic</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="Single_pb">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Single</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
